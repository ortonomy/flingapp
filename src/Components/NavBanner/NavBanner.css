.NavBanner {
  height: 4em;
  display: flex;
  flex-flow: row nowrap;
  justify-content: space-between;
  align-items: center;
}

.NavBanner img {
  max-height: 100%;
  max-width: 100%;
}
