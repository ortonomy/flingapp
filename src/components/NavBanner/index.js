import React, { Component } from 'react';
import { connect } from 'react-redux';
import Avatar from '../Avatar';
import banner from './NavBanner.module.css';
import logo from '../../assets/images/web-title.png';

class NavBanner extends Component {

  render() {
    return (
      <div className={banner.NavBanner}>
        <img src={logo} alt="Fling.App"/>
        <Avatar user={this.props.user} />
      </div>
    )
  }
}

export default NavBanner;
