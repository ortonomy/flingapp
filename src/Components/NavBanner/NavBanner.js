import React, { Component } from 'react';
import Avatar from '../Avatar/Avatar';
import './NavBanner.css';
import logo from '../../assets/images/full_title_compact_trans.png';

class NavBanner extends Component {
  render() {
    return (
      <div className="NavBanner">
        <img src={logo} alt="Fling.App"/>
        <Avatar />
      </div>
    )
  }
}

export default NavBanner;
