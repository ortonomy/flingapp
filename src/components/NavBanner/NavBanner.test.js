import React from 'react';
import NavBanner from '../NavBanner';

it('renders without crashing', () => {
  const wrapper = shallow(<NavBanner />);
  expect(wrapper).toMatchSnapshot();
});
