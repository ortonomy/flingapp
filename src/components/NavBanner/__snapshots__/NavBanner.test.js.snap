// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBanner />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="Fling.App"
          src="web-title.png"
/>,
        <Avatar
          user={undefined}
/>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Fling.App",
          "src": "web-title.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "user": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="Fling.App"
            src="web-title.png"
/>,
          <Avatar
            user={undefined}
/>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Fling.App",
            "src": "web-title.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "user": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
